servicemesh-install/topics/document-attributes.adoc

```diff
@@ -0,0 +1,39 @@
+// Standard document attributes to be used in the documentation
```

Comment:
This isn't a topic, it's a master document attributes file.  (As we discussed elsewhere, I'm used to having an external file rather than attributes in the headings)